packml_msgs/msg/Stats.msg

```diff
@@ -0,0 +1,29 @@
+# This message contains machine agnostic information that is extracted from PackML
+# standard states
+
+std_msgs/Header stamp       # Duration of statistics calculation
```

Comment:
Could you elaborate this a bit more? How would you see a `Header` being used to encode the "duration of .." anything?